libs/pages/environments/src/lib/ui/create-clone-environment-modal/create-clone-environment-modal.tsx

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable-next-line */
+import { EnvironmentModeEnum } from 'qovery-typescript-axios'
+import { FormEvent, useEffect, useState } from 'react'
+import { Controller, useFormContext } from 'react-hook-form'
+import { ClusterEntity, EnvironmentEntity, Value } from '@console/shared/interfaces'
+import { Button, ButtonStyle, InputSelect, InputText } from '@console/shared/ui'
+
+export interface CreateCloneEnvironmentModalProps {
+  onSubmit: () => void
+  environmentToClone?: EnvironmentEntity
+  clusters?: ClusterEntity[]
+  closeModal: () => void
+  loading: boolean
+}
+
+export function CreateCloneEnvironmentModal(props: CreateCloneEnvironmentModalProps) {
+  const { formState, control } = useFormContext()
+
+  const [environmentModes] = useState<Value[]>([
+    { value: 'automatic', label: 'Automatic' },
+    { value: EnvironmentModeEnum.PREVIEW, label: 'Preview' },
```

Comment:
We need to remove it, we can't create with a Preview mode

Reply:
Confirmed with Melvin, there is a problem with the EnvironmentModeEnum that should not propose PREVIEW. It's a wrong copy paste inside the swagger api